DEV-INTRO.md

```diff
@@ -78,70 +72,57 @@ The steps in this section only need to be done the first time you start working
 
 #### .env file
 
-You can copy the [`docker/env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
-
-For Linux/MacOSX
+You can copy the [`docker/env.example`](docker/env.example) file as .env in your pecan folder.
 
 ```sh
 cp docker/env.example .env
 ```
 
-For Windows
 
-```sh
-copy docker/env.example .env
-```
+The variables we want to modify are:
 
-- `COMPOSE_PROJECT_NAME` set this to pecan, the prefix for all containers
-- `PECAN_VERSION` set this to develop, the docker image we start with
+- `COMPOSE_PROJECT_NAME`, the prefix for all containers. Set this to "pecan".
+- `PECAN_VERSION`, the docker image we start with. Set this to "develop".
 
-Both of these variables should also be uncommented by removing the # preceding them.  At the end you should see the following if you run the following command  `egrep -v '^(#|$)' .env`. If you have a windows system, you will need to set the variable PWD as well, and for linux you will need to set UID and GID (for rstudio).
+Both of these variables should also be uncommented by removing the # preceding them.
 
-For Linux
+At the end you should see the following if you run the command  `egrep -v '^(#|$)' .env`:
 
 ```sh
-echo "COMPOSE_PROJECT_NAME=pecan" >> .env
-echo "PECAN_VERSION=develop" >> .env
-echo "UID=$(id -u)" >> .env
-echo "GID=$(id -g)" >> .env
+COMPOSE_PROJECT_NAME=pecan
+PECAN_VERSION=develop
 ```
 
-For MacOSX
+If you have a Linux system you will need to set UID and GID (these are needed by rstudio when sharing files between host and container):
```

Comment:
Is this actually Linux-specific? I have UID and GID set on my Mac and have not been able to get it working without. 
Should we recommend this step for every OS?